custodial-copy-confirmer/src/main/scala/uk/gov/nationalarchives/confirmer/Main.scala

```diff
@@ -70,9 +67,11 @@ object Main extends IOApp {
       _ <- logger.info(s"Processing message refs ${messages.map(_.message.ioRef).mkString(",")}")
       _ <- messages.parTraverse { sqsMessage =>
         val message = sqsMessage.message
-        val request = DADynamoDbRequest(message.dynamoTable, message.primaryKey, message.toUpdate)
-        IO.whenA(ocfl.checkObjectExists(message.ioRef))(dynamoClient.updateAttributeValues(request).void) >>
-          sqsClient.deleteMessage(config.sqsUrl, sqsMessage.receiptHandle)
+        val request = DADynamoDbRequest(config.dynamoTableName, message.primaryKey, Map(config.dynamoAttributeName -> "true".toAttributeValue.some))
```

Comment:
Could `"true".toAttributeValue.some` be a global/local (outside of `parTraverse`) variable?

Reply:
Could this value be a Boolean?

Reply:
It could but I don't think it adds much, it's easy to see this way that we're writing "true".
We're using the String "true" in other places in the files table. I can't remember why but this keeps it consistent.